corehq/apps/geospatial/utils.py

```diff
@@ -223,15 +229,42 @@ class CeleryTaskTracker(object):
 
     def __init__(self, task_key):
         self.task_key = task_key
+        self.progress_key = f'{task_key}_progress'
+        self.message_key = f'{task_key}_message'
```

Comment:
nit: looks like we are no longer using this after refactoring.

Reply:
Ah, I accidentally forgot this in when doing testing for the refactor. Nice catch, will remove.

Reply:
Addressed in [a4c7bb5](https://github.com/dimagi/commcare-hq/pull/35126/commits/a4c7bb5d50e939b2b55322d2e4df44ce96a57bbc).